README.md

```diff
@@ -1,23 +1,32 @@
-# Heiko
-Heiko rewritten in go! 
+Heiko
+=====
```

Comment:
Is there a specific reason for using this syntax instead of `# `? I think more people are familiar with the previous syntax.

Reply:
Ah, my bad this is because of my markdown format extension. I do prefer the underline syntax because it looks cleaner and more readable when looking at raw markdown. Do you want me to revert them?

Reply:
Reverted!

Reply:
Oh I didn't think of it that way, it makes sense.